lang_java/parsing/parser_java.mly

```diff
@@ -285,8 +289,7 @@ sgrep_spatch_pattern:
 
 item_no_dots:
  | statement_no_dots { Init (false, $1) }
- /*(* TODO: add class_member_declaration, and its method_declaration here,
-    * but many conflicts *)*/
+ | declaration { $1 }
```

Comment:
This is good, but you've introduced 3 new shift/reduce conflicts in the grammar.
This is fine, because it's hard to fix in general, and in many cases it actually works, but it's important to remember
that we should at some point do it the right way and fix those new introduced conflicts, and it's very important
to state which modifications added those conflicts (it's easier to fix when you know which diff introduced the conflict).

Reply:
@aryx I am happy to revert this, how does one fix the grammar shift/reduce conflicts? I did not see it during compilation! Maybe we should add a step so that we hard block it!